hedera-node/hedera-app/src/xtest/java/common/BaseScaffoldingModule.java

```diff
@@ -216,6 +220,12 @@ static ConfigProvider provideConfigProvider(@NonNull final Configuration configu
         return () -> new VersionedConfigImpl(configuration, 1L);
     }
 
+    @Provides
+    @Singleton
+    static ServicesRegistry provideServicesRegistry(@NonNull final ServicesRegistry servicesRegistry) {
```

Comment:
⚠️ If Dagger wanted a provider for a `ServicesRegistry`, we probably want to add a component builder method that `@BindsInstance` the `ServicesRegistryImpl` that we construct "by hand" at the beginning of `Hedera#main()`. 

(As written, I am worried the `ServicesRegistry` will not actually be a singleton in the system; but instead a singleton in the Dagger component **plus** the instance constructed by hand...this could be a problem.)

Reply:
Added this Michael. Does that work ?
```
    @Binds
    @Singleton
    ServicesRegistry bindServicesRegistry(ServicesRegistryImpl servicesRegistry);
```